osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr52.json

```diff
@@ -162,5 +162,25 @@
         "access": "R"
       }
     ]
-  }]
+  },
+    {
+      "tag": "ALARM_REGISTER_2",
+      "description": "Alarm register 2",
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.97.98.1.255",
```

Comment:
Ik 0.0.97.98.1.255 zie deze niet in de 5.2 specs terug komen, zou best kunnen kloppen, maar wil graag weten waar deze staan in de specs

Reply:
Goed punt, we gaan dit even uitzoeken.

Reply:
This is specified in an addendum of spec SMR 5.2 and is added to the wiki.

Reply:
Check, good that it has been added